packages/client-discord/src/actions/joinvoice.ts

```diff
@@ -115,31 +113,30 @@ export default {
             );
         });
 
-        if (!state.voiceManager) {
-            state.voiceManager = new VoiceManager({
-                client: state.discordClient,
-                runtime: runtime,
-            });
-        }
```

Comment:
I suppose we don't need this check anymore...

Reply:
yea It seems that this code is outdated: https://github.com/elizaOS/eliza/blob/b0e665edec6943857db9fea36800c6812d392d18/packages/core/src/types.ts#L271